### ISSUE TITLE
question: [Evaluation results of the Q8BERT]

### ISSUE THREAD

**Issue by Shinonome233** [IntelLabs/nlp-architect#202]
Hi, I read the paper "Q8BERT, Quantized 8Bit BERT". and it says that the results of Q8BERT are averaged after 5 experiments. But I have conducted several fine-tuning experiments on MRPC benchmark, but found that the results are the same every time. 

In addition, in the fine-tuning process, inference and evaluation will be performed every 50 steps of training. Is your result the best one you adopted during training, or the last one? They look very different.

**Comment by peteriz** [comment]
@ofirzaf

**Comment by ofirzaf** [comment]
1. If you get the same result every time you probably use the same seed in all of your experiments. Please use the `--seed` flag to change the seed for each experiment which defaults to `42`.
2. The evaluation results we reported are using the resulted model at the end of fine-tuning, not the best during fine-tuning.
